src/Adapter/Cart/CartPresenter.php

```diff
@@ -273,15 +273,17 @@ public function present($cart)
             );
         }
 
-        $shipping_cost = $cart->getTotalShippingCost(null, $this->includeTaxes());
-        $subtotals['shipping'] = array(
-            'type' => 'shipping',
-            'label' => $this->translator->trans('Shipping', array(), 'Shop.Theme.Checkout'),
-            'amount' => $shipping_cost,
-            'value' => $shipping_cost != 0
-                ? $this->priceFormatter->format($shipping_cost)
-                : $this->translator->trans('Free', array(), 'Shop.Theme.Checkout'),
-        );
+        if (!$cart->isVirtualCart()) {
+            $shipping_cost = $cart->getTotalShippingCost(null, $this->includeTaxes());
```

Comment:
would you mind to update the variable name to camelcase form ? ``$shippingCost``